File: baekjoon_python/oxQuiz.py
num = int(input())
subScore = 0
score = 0
for i in range(0, num) :
    str = input()
    for j in range(0, len(str)) :
        if(str[j] == 'O') :
            subScore += 1
            score += subScore
        else :
            subScore = 0
    print(score)
    score = 0
    subScore = 0

File: baekjoon_java/Oxquiz.java
package baekjoon_java;
import java.util.*;
//"OOXXOXXOOO"의 점수는 1+2+0+0+1+0+0+1+2+3 = 10점
//1차원 배열사용
public class Oxquiz {
	public static void quiz(String s) {
		int plus = 1;
		int score = 0;
		String[] arr = s.split("");
		for(int i = 0; i < s.length(); i++) {
			if(arr[i].equals("O")) {
				score += plus;
				plus++;
			}
			if(arr[i].equals("X")) {
				plus = 1;
			}
		}
		System.out.println(score);
	}
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int num = input.nextInt();
		for(int i = 0; i < num; i++) {
			String s = input.next();
			quiz(s);
		}
	}
}
